### PR TITLE
Added conversations.list endpoint functionality and cursor request parameter functionality in base response class.

### PR DESCRIPTION
**Pull request by jasonproulxFIRST**
Resolves Issue #242 
Resolves original bug identified in Pull Request #231

### CODE REVIEW

**Review by alexherreid — APPROVED** [review]
FWIW, I grabbed these changes and they work as advertised with the new conversations endpoint as well as the issue that I attempted to fix in rejected PR #231

File: SlackAPI/SlackClient.cs
Comment (Inumedia) [inline SlackAPI/SlackClient.cs:158]: Can you fix the indentation in this method

**Review by Inumedia — COMMENTED** [review]
Hi, once the indentation is fixed I can merge this. As a bit meta, this API seems odd, kinda curious why `types` would be a comma delimited list rather than just a regular array?

File: SlackAPI/SlackClient.cs
Comment (jasonproulxFIRST) [reply SlackAPI/SlackClient.cs]: Indentation has been fixed. Thanks for pointing it out!

**Comment by jasonproulxFIRST** [comment]
The requested indentation issue was resolved. Are these changes good to go?

File: SlackAPI/SlackClient.cs
Comment (Inumedia) [inline SlackAPI/SlackClient.cs:157]: Just realized these aren't actually getting passed into the list of parameters

**Review by Inumedia — COMMENTED** [review]
Hi, thanks for reminding me to look at this again.

I double checked the changes and noticed one of the new calls creates the Tuples for the parameters but never actually adds it to the list. Once that's done this can be approved and will just need to wait for me to merge + deploy the new nuget package

File: SlackAPI/SlackClient.cs
Comment (jasonproulxFIRST) [reply SlackAPI/SlackClient.cs]: Silly mistake! Just updated this with the correct code.

**Review by Inumedia — APPROVED** [review]

**Pull request merged**
